app/src/main/java/com/g11x/checklistapp/data/Database.java

```diff
@@ -217,7 +227,7 @@ private static Uri createContentUri() {
     }
 
     private static String createCreateTableSql() {
-      return "create table " + TABLE_NAME + " (_ID integer primary key, " + TITLE_COLUMN + " text, " + MESSAGE_COLUMN + " text, " + READ_COLUMN + " boolean);";
+      return "create table " + TABLE_NAME + " (_ID integer primary key, " + TITLE_COLUMN + " text, " + MESSAGE_COLUMN + " text, " + READ_COLUMN + " boolean, " + SENT_TIME + " integer);";
```

Comment:
While this line is changing&hellip; we've got `Database.ID_COLUMN` for the `_ID` string.